meetings/2021-09-22.md

```diff
@@ -0,0 +1,159 @@
+* ...: On the other hand, ACP uses the notion of owner for default permissions (the owner of a resource can always edit its access control rules) (note that ACP also expects a server to be given the owners of a resource being accessed, it is also a Solid protocol thing).
+* ...: Obviously, it would be possible to define rules involving a creator and apply them by default (using the inheritance mechanism).
+* ...: See also the ACP [agent matcher](https://htmlpreview.github.io/?https://github.com/solid/authorization-panel/blob/initial-acp-spec/proposals/acp-specification/index.html#agent-matcher) concept described in the spec
+* **Henry**: To get back to Elf's point above, I want to point out that we now have 3 authrization systems being developed. WAC, ACP, and the interop panel. This seems like unecessary duplication of work.
+* ... The Interop panel work I guess is as I understand about limiting access to application. But as per [discussion 258: Use WAC ontology for authorizing authentication](https://github.com/solid/authorization-panel/discussions/258) we can use the WAC ontology to authorize app authentication. wac can be used to say this App can read these resources. For example one could have a rule that says: only authorize the bank app to be given signed headers for these three web sites.
+* **Barath**: ultimately systems work well when they are pluralistic. we don't need to produce a single answer.. we can't forsee all future Use Cases.. talking about diverse world might be using this thing.. what i'd like to see is.. it is OK to see different specs.. as long as there is loose interop. and people can migrate... web works through coexistence.. 
+* **Pavlik**: translating data grants into ACP rules. data grants give a way for auth agents or auth apps.. owned by different users.
+* **Kjetil**: purpose of the issue i opened is to establish baseline interop. By putting in the protocol spec's the various operations we see coming out of this.. on the greater issue of diversity of work: there is more of a balancing act. Unecessary divergence is  likely to hurt interop. on the other hand, being too restrictive may stop people from being creative and that is also bad. 
+* **Sarven**: +1 protocol spec as baseline.
+* **Sarven**: why isn't the data grants being translated to WAC?
+* **Pavlik**: EricP were looking into translating to WAC. wac-ucr had list of limitation found in that process.
+* **Sarven**: WAC is called basic access control ontology, it should be able to do a lot of stuff for you. There is long list of things the can also be done which it doesn't do - but it can via extensions.. or updating the spec.
+* ...: This didn't stop ACP to be considered, unless there are fundamental issues, we can't do greenfielding on everything.
```

Comment:
```suggestion
* **Sarven**: WAC.. ns/auth/acl is basic access control ontology, it should be able to do a lot of stuff for you. There is long list of things the can also be done which it doesn't do - but it can via extensions.. or updating the spec.
* ...: (*The scribe did not transcribe the context for this message but it was a continuation to above; while there is existing work, new work is welcome. the response is in context of operations, access controls, and making fundamental changes to the Solid Protocol re "greenfielding" as mentioned in the comments of the topic issue*) This didn't stop ACP to be considered, unless there are fundamental issues, we can't do greenfielding on everything.
```